xml/System.Windows/EventRoute.xml

```diff
@@ -1,224 +1,280 @@
+      </Parameters>
+      <Docs>
+        <param name="target">Specifies the target object of which the handler is to be added to the route.</param>
+        <param name="handler">Specifies the handler to be added to the route.</param>
+        <param name="handledEventsToo">Indicates whether or not the listener detects events that have already been handled.</param>
+        <summary>Adds the specified handler for the specified target to the route.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ It is not an error to add a handler for a particular target instance twice. In such a case, the handler will be called twice.  
+  
+ ]]></format>
+        </remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="PeekBranchNode">
+      <MemberSignature Language="C#" Value="public object PeekBranchNode ();" />
+      <MemberSignature Language="ILAsm" Value=".method public hidebysig instance object PeekBranchNode() cil managed" />
+      <MemberSignature Language="DocId" Value="M:System.Windows.EventRoute.PeekBranchNode" />
+      <MemberSignature Language="VB.NET" Value="Public Function PeekBranchNode () As Object" />
+      <MemberSignature Language="F#" Value="member this.PeekBranchNode : unit -&gt; obj" Usage="eventRoute.PeekBranchNode " />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; System::Object ^ PeekBranchNode();" />
+      <MemberType>Method</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>PresentationCore</AssemblyName>
+        <AssemblyVersion>3.0.0.0</AssemblyVersion>
+        <AssemblyVersion>4.0.0.0</AssemblyVersion>
+        <AssemblyVersion>5.0.0.0</AssemblyVersion>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <Attributes>
+        <Attribute>
+          <AttributeName Language="C#">[System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Advanced)]</AttributeName>
+          <AttributeName Language="F#">[&lt;System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Advanced)&gt;]</AttributeName>
+        </Attribute>
+      </Attributes>
+      <ReturnValue>
+        <ReturnType>System.Object</ReturnType>
+      </ReturnValue>
+      <Parameters />
+      <Docs>
+        <summary>Returns the top-most element on the event route stack at which two logical trees diverge.</summary>
+        <returns>The top-most element on the event route stack at which two logical trees diverge.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+If a node in the tree has different visual and logical,
+                FrameworkElement will store the node on this stack of
+                branches.  If the route ever returns to the same logical
+                tree, the event source will be restored.
+                <para />
+                NOTE: This method needs to be public because it is used
+                by FrameworkElement.

```

Comment:
Is this information relevant to the final user?

Reply:
I'll remove these.